debugging/README.md

```diff
@@ -39,3 +39,65 @@ Name | Sponsor | Protocol available | State |
 [Chrome DevTools]: https://github.com/ChromeDevTools/devtools-frontend
 [Theseus]: https://github.com/adobe-research/theseus
 
+
+## Blackbox
+
+Blackbox is a popular term that allows debugging user-land code without touching in their internals.
+
+For futher reference in nodejs debugging follow [these instructions](https://nodejs.org/en/docs/guides/debugging-getting-started/).
+
+### Enabling Blackbox through UI
+
+// TODO: waiting https://github.com/GoogleChrome/developer.chrome.com/issues/1074 resolution and point to this tutorial.
+
+### Enabling Blackbox through CLI
+
+Let's create a file called `example-blackbox.js` to simulate a blackbox usage
+
+```js
+module.exports = {
+  doSomethingAsync: async () => {
+    debugger
+    console.log('Async done')
+  }
+}
+```
+
+Then we have our `index.js` file
+
+```js
+const { doSomethingAsync } = require('./example-blackbox')
+async function main() {
+  await doSomethingAsync()
+  console.log('Done!')
+}
+
+main()
+```
+
+Start debugging using CLI:
+
+```console
+node inspect index.js
+```
+
+and enable the blackbox by calling [`Debugger.setBlackBoxPatterns`](https://chromedevtools.github.io/devtools-protocol/tot/Debugger/#method-setBlackboxPatterns) and passing the ignore regex, in our case a simple `example-blackbox.js`:
+
+```console
+debug> Debugger.setBlackboxPatterns({ patterns: ['example-blackbox.js'] })
+```
+
+Afterall, perform a `continue` and see the script be running without stopping into `example-blackbox.js` function.
+
+```console
+debug> continue
```

Comment:
nit to change to `cont` or `c` ?
https://nodejs.org/api/debugger.html#debugger_stepping

to avoid:
```
debug> continue
continue
^^^^^^^^

Uncaught SyntaxError: Illegal continue statement: no surrounding iteration statement
```

Reply:
You right